lineage/figures/figureS53.py

```diff
@@ -25,7 +26,7 @@ def makeFigure():
 
     # Get list of axis objects
     ax, f = getSetup((10, 10), (3, 3))
-    lin_params = {"pi": pi, "T": T, "E": E2, "desired_num_cells": min_desired_num_cells}
+    lin_params = {"pi": pi, "T": T, "E": E2}
     number_of_columns = 25
     figureMaker3(ax, *accuracy(lin_params, number_of_columns))
```

Comment:
I would delete the argument `lin_params` argument.
```suggestion
    figureMaker3(ax, *accuracy(number_of_columns))
```